tests/neuro_san/client/test_math_guy.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+from unittest import TestCase
+
+from tests.neuro_san.client.data_driven_agent_test import DataDrivenAgentTest
+from tests.neuro_san.client.unit_test_assert_forwarder import UnitTestAssertForwarder
+
+
+class TestMathGuy(TestCase):
+    """
+    Tests basic functionality via the math_guy agent.
+    """
```

Comment:
Add two new tests that attack sly_data.

Reply:
I don't think that's sufficient. It doesn't test that:
- sly_data is passed by the client to the agent server
- that the sly_data is still available in a follow up message
These tests can be added later.

Reply:
1. It does indeed test that sly data is passed to the system. This is the x and y of the test. At the moment, the unit tests are only going direct because there is no server stood up for unit tests at the moment.
2. Agreed